### PR TITLE
NMS-12650: Restart documentation

### PR DESCRIPTION
**Pull request by Bonrob2**
Restart/shutdown documentation
Updated thresholding to address formatting issues.

### All Contributors

* [x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [x] Have you made a comment in that issue which points back to this PR?
* [x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [x] If this is a new or updated feature, is there documentation for the new behavior?
* [x] If this is new code, are there unit and/or integration tests?
* [x] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

Once the reviewer(s) accept the PR and the branch passes continuous integration in Bamboo, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-12650

### CODE REVIEW

**Comment by Bonrob2** [comment]
Updated restart procedure as per feedback from Jesse. Updated Grafana Dashboards as PDF docs due to missing content, updated formatting of Thresholding sections, as the heading levels were incorrect and were creating problems.

**Comment by Bonrob2** [comment]
@indigo423 @j-white I'd like to merge this documentation so it's available in next week's release.

File: opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc [hunk 1]
```diff
@@ -278,3 +278,73 @@ WARNING: If `org.opennms.netmgt.jasper.measurement.ssl.enable` is set to `false`
...
+Separate multiple recipient emails with a comma. 
+.. Webhook allows you to post the generated report to the specified URL. 
+. Click `Deliver Report`.
+. To schedule the report for regular delivery, click `Schedule this report`.
+. Specify the report frequency (daily, days per week, etc.) and interval of the report.
+. Click `Schedule Report`.
+
+Scheduled reports appear in the `Report Schedules` tab, where you can edit or delete them:
+
+image::database-reports/PDF-Report.png[pdf report, 600]
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc:350]: I couldn't find this image here

Comment (Bonrob2) [reply opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc:350]: > I couldn't find this image here
Added image.

**Pull request merged**